src/metabase/api/dashboard.clj

```diff
@@ -657,6 +657,8 @@
    ;; tabs should be required in production, making it optional because lots of
    ;; e2e tests curerntly doesn't include it
    tabs [:maybe (ms/maps-with-unique-key [:sequential UpdatedDashboardTab] :id)]}
+  (log/warn
+   (trs "DELETE /api/dashboard/:id/cards is deprecated. Use PUT /api/dashboard/:id instead."))
```

Comment:
From now on we don't want to translation for log https://github.com/metabase/metabase/issues/10764
```suggestion
   "DELETE /api/dashboard/:id/cards is deprecated. Use PUT /api/dashboard/:id instead.")
```